### PR TITLE
RecSys - Improve caching

### PR DESCRIPTION
**Pull request by adrian-musora**
MWP PR: https://github.com/railroadmedia/musora-web-platform/pull/846

### CODE REVIEW

File: src/Services/ContentService.php [hunk 1]
```diff
@@ -47,89 +47,6 @@
 
 class ContentService
 {
-    /**
-     * @var ContentRepository
-     */
-    private $contentRepository;
```
Comment (roxanariza) [inline src/Services/ContentService.php:53]: Please revert these updates(line 50 - line 132). All the catalogs and other pages are crashing now

File: src/Services/ContentService.php [hunk 2]
```diff
@@ -143,63 +60,34 @@ class ContentService
...
+        private UserContentProgressRepository $userContentProgressRepository,
+        private ContentTopicRepository $contentTopicRepository,
+        private ContentInstructorRepository $contentInstructorRepository,
+        private ContentStyleRepository $contentStyleRepository,
+        private ContentBpmRepository $contentBpmRepository,
+        private DatabaseManager $databaseManager,
+        private ContentVideoRepository $contentVideoRepository,
+        private RecommendationService $recommendationService,
     ) {
-        $this->contentRepository = $contentRepository;
```
Comment (roxanariza) [inline src/Services/ContentService.php:184]: Revert please

**Review by roxanariza — CHANGES_REQUESTED** [review]

File: src/Services/ContentService.php (hunk 1 above)
Comment (adrian-musora) [reply src/Services/ContentService.php:53]: Merged with 2.0- and crashes are resolved.

Comment (roxanariza) [reply src/Services/ContentService.php:53]: Issues are fixed now! Thanks!

File: src/Services/ContentService.php (hunk 2 above)
Comment (roxanariza) [reply src/Services/ContentService.php:184]: Issues are fixed now! Thanks!

**Review by roxanariza — APPROVED** [review]

**Pull request merged**